impl/client/src/main/javascript/web/dojo/pentaho/common/propertiesPanel/Panel.js

```diff
@@ -1487,29 +1488,66 @@ define([
 
   var SliderUI = declare([HorizontalSlider, StatefulUI, Evented], {
 
-      className: "propPanel_slider propPanel_control",
-      minimum: 0,
-      maximum: 100,
-      style: "width: 100%",
-      intermediateChanges: true,
-      discreteValues: true,
+    className: "propPanel_slider propPanel_control",
+    minimum: 0,
+    maximum: 100,
+    style: "width: 100%",
+    intermediateChanges: true,
+    discreteValues: true,
 
-      constructor: function(options) {
+    constructor: function (options) {
 
-        options.id = newId(this.model.id + "_slider"); // -> this.id
+      options.id = newId(this.model.id + "_slider"); // -> this.id
 
-        this.value = this.model.value;
+      this.value = this.model.value;
+
+      if (this.model.minimum) {
+        this.minimum = this.model.minimum;
+      }
+      if (this.model.maximum) {
+        this.maximum = this.model.maximum;
+      }
 
-        if(this.model.minimum) { this.minimum = this.model.minimum; }
-        if(this.model.maximum) { this.maximum = this.model.maximum; }
+      this.discreteValues = this.maximum - this.minimum + 1;
+    },
 
-        this.discreteValues = this.maximum - this.minimum + 1;
-      },
+    postCreate: function () {
 
-      onChange: function() {
-        this.model.set('value', this.value);
-      }
-    });
+      this.slider = registry.byId(this.id);
+
+      this.own(
+        on(this.domNode, "mouseover", lang.hitch(this, "onMouseOver")),
+        on(this.domNode, "mouseleave", lang.hitch(this, "hideTooltip")),
+        on(this.domNode, "focusin", lang.hitch(this, "onFocusIn")),
+        on(this.domNode, "focusout", lang.hitch(this, "hideTooltip"))
+      );
+
+      this.uiKeyNode = this.slider.focusNode;
+      this.inherited(arguments);
+    },
+
+    destroy: function () {
+      this.inherited(arguments);
+      this._startupWidgets = null;
+      this._supportingWidgets = null;
```

Comment:
Are these really used?

Reply:
To be honest, I was following the other 'destroy' methods in the same file.

Reply:
Doesn't seem to apply to the slider.